boskos/janitor/Dockerfile

```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.5
+FROM gcr.io/k8s-testimages/gcloud-in-go:0.1
 MAINTAINER senlu@google.com
 
-ADD ["janitor", "/"]
+ADD ["janitor", \
+     "https://raw.githubusercontent.com/kubernetes/test-infra/master/jenkins/janitor.py", \
```

Comment:
How about moving janitor.py into this file? I think we want to copy this file from the checked out version of test-infra, not master on github